src/components/goToOrdersButton.tsx

```diff
@@ -3,14 +3,23 @@ import { AppButton } from "./appButton";
 import { useDispatch, useSelector } from "react-redux";
 import { showOrdersButtonSelector } from "../store/appControl/selectors";
 import { updateCurrentScene } from "../store/appControl/appControlSlice";
+import { fetchOrders } from "../store/orders/ordersSlice";
+import { AppDispatch } from "../store/store";
+import { orderStatusSelector } from "../store/orders/selectors";
 
 export function GoToOrdersButton() {
-  const dispatch = useDispatch();
+  const dispatch = useDispatch<AppDispatch>();
+  const orderStatus = useSelector(orderStatusSelector);
 
   const showOrdersButton = useSelector(showOrdersButtonSelector);
 
-  const handleClickViewOrders = () => {
-    dispatch(updateCurrentScene("ORDER"));
+  const handleClickViewOrders = async () => {
+    await dispatch(fetchOrders());
```

Comment:
dispatch action is not an async one, so you cant await it

Reply:
But when I delete await it works uncorrectly. 
`const handleClickPlaceOrder = async () => {
    await dispatch(createOrder());
    if (orderStatus === "completed") {
      dispatch(fetchOrders());
      dispatch(updateCurrentScene("SUCCESS"));
    } else if (orderStatus === "failed") {
      console.error("Something went wrong with an order upload.");
    }
  };`
If I place first order without await it fetches nothing in the next line (fetchOrders()) and it causes it's impossible to see any orders, because GoToOrdersButton is visible when orders.length > 0. With "await" everything is OK!